source/test/F0.Minesweeper.Components.Tests/Pages/Game/GameTests.razor

```diff
@@ -75,25 +93,32 @@
 			context.ComponentFactories.AddStub<GameEndStatus>();
 
 			changedEvent = new();
+			restartGameEvent = new();
 
 			Mock<IEventAggregator> eventAggregatorMock = new();
 			eventAggregatorMock
 				.Setup(ea => ea.GetEvent<DifficultyLevelChangedEvent>())
 				.Returns(changedEvent);
 
+			eventAggregatorMock
+				.Setup(ea => ea.GetEvent<RestartGameEvent>())
+				.Returns(restartGameEvent);
+
 			context.Services.AddSingleton(eventAggregatorMock.Object);
 
 			// Act
 			using(IRenderedFragment gameFragment =
 				context.Render(
 @<Game />
-				)
+	)
```

Comment:
I think the indentation here isn't quite right.

Reply:
I fixed that :)